CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 1.23.0
+ * added `MYSQLI_REFRESH_REPLICA` constant to PHP 8.1 polyfill
```

Comment:
```suggestion

 * Add `MYSQLI_REFRESH_REPLICA` constant to PHP 8.1 polyfill
```

Reply:
This file uses both past and present tense so I just followed convention from `1.22.0`. @nicolas-grekas what do you think?